src/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import Counter from './pages/counter';
```

Comment:
```suggestion
import React, { useState } from 'react';
import ReactDOM from 'react-dom';

import Counter from './pages/counter';
```
리액트 라이브러리 import와 로컬 컴포넌트 import 를 구분하기 위해 한 줄 공백을 주시면 좋을 것 같습니다~

지금은 모듈이 별로 없지만, 모듈 간에도 역할이 있고, 분리를 해두는 습관을 두면 좋을 것 같습니다 :)